### PR TITLE
Amplitude : Adding device_brand in mapping

### PR DESCRIPTION
**Pull request by abandonned**
## Description of the change
Adding device_brand in mapping jsons.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: v0/destinations/am/utils.js [hunk 1]
```diff
@@ -61,10 +61,20 @@ function getPlatform(payload, sourceKey) {
     : payloadVal;
 }
 
+function getBrand(payload, sourceKey) {
+  const payloadVal = get(payload, sourceKey);
+
+  if (payload.channel && payload.channel.toLowerCase() === "web") {
+    return getInfoFromUA("device.brand", payload, payloadVal);
```
Comment (saikumarrs) [inline v0/destinations/am/utils.js:68]: Do we know for sure, this information will be part of UA?

Comment (utsabc) [reply v0/destinations/am/utils.js:68]: @abandonned AFAIK `ua` will not contain the brand information so we can skip this part. Even for web source payload, we have to have the device brand info in `context.device`

Lets make it clear in our docs what AM takes for computing `device_family` and from where we are picking those components from our payload

**Comment by utsabc** [comment]
@abandonned please add test cases around this as well.

File: v0/destinations/am/utils.js (hunk 1 above)
Comment (abandonned) [reply v0/destinations/am/utils.js]: Updated as per latest discussion. @utsabc @saikumarrs

**Review by utsabc — DISMISSED** [review]

File: v0/destinations/am/utils.js [hunk 2]
```diff
@@ -61,10 +61,19 @@ function getPlatform(payload, sourceKey) {
     : payloadVal;
 }
 
+// eslint-disable-next-line consistent-return
```
Comment (saikumarrs) [inline v0/destinations/am/utils.js:64]: Return `undefined` by default and remove this comment

**Review by saikumarrs — CHANGES_REQUESTED** [review]

File: v0/destinations/am/utils.js (hunk 2 above)
Comment (abandonned) [reply v0/destinations/am/utils.js]: updated

**Review by saikumarrs — APPROVED** [review]

**Review by utsabc — APPROVED** [review]

**Pull request merged**